planning/autoware_static_centerline_generator/src/static_centerline_generator_node.cpp

```diff
@@ -745,6 +829,92 @@ void StaticCenterlineGeneratorNode::connect_centerline_to_lanelet()
   }
 }
 
+JitterDetectionResult StaticCenterlineGeneratorNode::detect_jitter_sections(
+  const std::vector<TrajectoryPoint> & centerline)
+{
+  JitterDetectionResult result;
+
+  if (centerline.size() < 4) {
+    return result;  // Need at least 4 points for jitter analysis
+  }
+
+  std::vector<double> angle_diffs;
+  const int64_t jitter_deg_threshold =
+    autoware::universe_utils::getOrDeclareParameter<int64_t>(*this, "jitter_deg_threshold");
+  const double jitter_rad_threshold = jitter_deg_threshold * M_PI / 180.0;
+
+  // Detect centerline jitter from angle differences
+  auto normalize_angle = [](double angle) {
+    while (angle > M_PI) angle -= 2.0 * M_PI;
+    while (angle < -M_PI) angle += 2.0 * M_PI;
+    return angle;
+  };
```

Comment:
can you use `autoware_utils_math::normalize_radian`?

https://github.com/autowarefoundation/autoware_utils/blob/3b0094fa0f4455e63418fb1f86b338e3f7578644/autoware_utils_math/include/autoware_utils_math/normalization.hpp#L36-L46